doc/nrf/gs_ins_windows.rst

```diff
@@ -340,26 +340,59 @@ Complete the following steps to set up the |SES| environment:
 
        Open nRF Connect SDK Project menu
 
-#. Set the Zephyr Base directory to the full path to ``ncs\zephyr``.
-   The GNU ARM Embedded Toolchain directory is the directory where you installed the toolchain (for example, ``c:\gnuarmemb``).
+#. Set the Zephyr Base directory to the full path to ``ncs/zephyr``.
+   The GNU ARM Embedded Toolchain directory is the directory where you installed the toolchain (for example, ``c:/gnuarmemb``).
 
     .. figure:: images/ses_notset.png
        :alt: Zephyr Base Not Set prompt
 
        Zephyr Base Not Set prompt
 
+.. buildenv_end
+
+.. buildenv_path_start
+
+4. Make sure the locations of tools are added to the PATH variable.
+   On Windows and Linux, SES uses the PATH variable to find executables.
```

Comment:
Only those executables that are not explicitly configured, see my other comment.